src/components/views/ProductList/ProductList.js

```diff
@@ -1,9 +1,48 @@
 import React from 'react';
-// import PropTypes from 'prop-types';
+import PropTypes from 'prop-types';
 import styles from './ProductList.module.scss';
+import ProductBox from '../../common/ProductBox/ProductBox';
 
-const ProductList = () => <div className={styles.root}>This is ProductList</div>;
+const ProductList = ({ category, products }) => {
+  return (
```

Comment:
Brakuje przekierowania na stronę Not Found, jeżeli kategoria nie zostąła znaleziona